src/components/footer.js

```diff
@@ -80,10 +76,26 @@ const Footer = () => (
         </FooterLink>
         <FooterLink href="https://createrefresh.eu/faq/">FAQ</FooterLink>
       </div>
-      <License className="dn-ns center">
-        Except where otherwise noted, content on this site is licensed under a
-        Creative Commons Attribution-NonCommercial 4.0 International (CC BY-NC
-        4.0)
+      <License className="center">
+        <p className="mt1 dn-ns db">
+          Except where otherwise noted, content on this site is licensed under a
+          Creative Commons Attribution-NonCommercial 4.0 International (CC BY-NC
+          4.0)
+        </p>
+        <p className="mt3 mt2-ns font-7-ns tr-ns">
+          Built by{" "}
+          <a className="color-inherit link" href="https://www.infactcoop.com/">
+            inFact Digital Co-Op
```

Comment:
capital 'i' 👍 and I know it's not in the designs but do you think we could get away with underlining the links?